sdk/python/tests/test_util.py

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import shutil
+import zipfile
+import yaml
+import tempfile
+import kfp_tekton.compiler as compiler
+import filecmp
+
+def _get_yaml_from_zip(zip_file):
+    with zipfile.ZipFile(zip_file, 'r') as zip:
+        with open(zip.extract(zip.namelist()[0]), 'r') as yaml_file:
+            return list(yaml.safe_load_all(yaml_file))
+
+
+def test_basic_workflow_without_decorator(test_data_dir):
+    """Test compiling a workflow and appending pipeline params."""
+
+    sys.path.append(test_data_dir)
+    import basic_no_decorator
+    try:
+        compiled_workflow = compiler.TektonCompiler()._create_workflow(
+            basic_no_decorator.save_most_frequent_word,
+            'Save Most Frequent',
+            'Get Most Frequent Word and Save to GCS',
+            [
+                basic_no_decorator.message_param,
+                basic_no_decorator.output_path_param
+            ])
+        print("SUCCESS: basic_no_decorator.py")
+    except:
+        print("FAILURE: basic_no_decorator.py")
+
+def test_composing_workflow(test_data_dir):
```

Comment:
Can we generalize this helper function to take any nested pipeline? We also have few nested pipeline in the sample repository that we want to make it pass. You can add arguments to also take the class name and the pipeline function names.